cmd/minikube/cmd/root.go

```diff
@@ -146,26 +139,21 @@ func usageTemplate() string {
 `, translate.T("Usage"), translate.T("Aliases"), translate.T("Examples"), translate.T("Available Commands"), translate.T("Flags"), translate.T("Global Flags"), translate.T("Additional help topics"), translate.T(`Use "{{.CommandPath}} [command] --help" for more information about a command.`))
 }
 
-// Handle config values for flags used in external packages (e.g. glog)
-// by setting them directly, using values from viper when not passed in as args
-func setFlagsUsingViper() {
-	for _, config := range []string{"alsologtostderr", "log_dir", "v"} {
```

Comment:
This was also deleted, which may be related to the alsologtostderr bug?

Reply:
I encouraged the author to remove this crazy bit of undocumented functionality: storing config values for v/alsologtostderr/log_dir didn't seem worth the extra code to support it.

(It's been 4 years since introduction and I've never seen it acknowledged as a feature)